docs/sql/statements/create-user.rst

```diff
@@ -51,4 +52,15 @@ Clauses
 The following ``user_parameter`` are supported to define a new user account:
 
 :password:
-  The password as cleartext entered as string literal.
+  The password as cleartext entered as string literal. e.g.::
+
+     CREATE USER john WITH (password='foo')
+
+  ::
+
+     CREATE USER john WITH password='foo'
+
+  ::
+
+     CREATE OK, 1 row affected (... sec)
```

Comment:
AFAIK we also allow space separated password (https://github.com/crate/crate/pull/15041/files#diff-79f6582d7c01256716a6ed5cd1e4ef4da0737a057b6c45ffb03b94f30fe85eaaR864), can we add such example(s)?
ie space separated, including/non-including WITH

Reply:
yes, sorry, I had those, but I messed something up while rebasing with master, pushed updated version.